### PR TITLE
[stable/prometheus] Add extraHostPathMounts config

### PR DESCRIPTION
**Pull request by yuanying**
* Add nodeExporter.extraHostPathMounts to prometheus

### CODE REVIEW

**Comment by yuanying** [comment]
Hi, I want to use [textfile collector](https://github.com/prometheus/node_exporter#textfile-collector) for node-exporter. But currently it seems that there are no way to share textfile metrics data between other pod which collect another metrics.

So, how about to add extraMounts option to node-exporter to share such data?

**Comment by mgoodness** [comment]
@k8s-bot ok to test

File: stable/prometheus/values.yaml [hunk 1]
```diff
@@ -214,6 +214,14 @@ nodeExporter:
   ##
   extraArgs: {}
 
+  ## Additional node-exporter extra hostPath mounts
+  ##
+  extraHostPathMounts: []
```
Comment (mgoodness) [inline stable/prometheus/values.yaml]: Let's get this array documented in the chart's README.

File: stable/prometheus/values.yaml [hunk 2]
```diff
@@ -271,6 +279,14 @@ server:
   ##
   extraArgs: {}
 
+  ## Additional server extra hostPath mounts
+  ##
+  extraHostPathMounts: []
```
Comment (mgoodness) [inline stable/prometheus/values.yaml]: Needs to be added to the chart's README.

**Review by mgoodness — CHANGES_REQUESTED** [review]

File: stable/prometheus/values.yaml (hunk 1 above)
Comment (yuanying) [reply stable/prometheus/values.yaml]: Thanks for reviewing.
I have added document to README.

**Review by mgoodness — APPROVED** [review]

**Pull request merged**